exporter/containerimage/export.go

```diff
@@ -366,16 +366,13 @@ func (e *imageExporterInstance) unpackImage(ctx context.Context, img images.Imag
 		return err
 	}
 
-	topLayerRef := src.Ref
-	if len(src.Refs) > 0 {
-		if r, ok := src.Refs[defaultPlatform()]; ok {
-			topLayerRef = r
-		} else {
-			return errors.Errorf("no reference for default platform %s", defaultPlatform())
-		}
+	ref, ok := src.FindRef(defaultPlatform())
```

Comment:
This needs to be moved above the `images.Manifest` or the `NotFound` error from `images.Manifest` should also be ignored.

Reply:
Good catch thanks! I've moved this block to the top of the function.